### PR TITLE
chore: minor fix

### PR DESCRIPTION
**Pull request by KeisukeShima**
Related https://github.com/tier4/autoware-spell-check-dict/pull/256

I have made some minor corrections.

### CODE REVIEW

File: tests/test_sort_package_xml.py [hunk 1]
```diff
@@ -14,12 +13,12 @@
...
     ],
 )
-def test(case: str, datadir: Path):
+def test(case: str, datadir: Path, tmp_path: Path):
     input_file = datadir / f"{case}.input.xml"
     answer_file = datadir / f"{case}.answer.xml"
 
     # Create a temporary file to prevent overwriting original files
-    tmp_file = Path(NamedTemporaryFile(mode="w", suffix=".xml").name)
+    tmp_file = tmp_path / f"{case}.input.xml"
```
Comment (kenji-miyake) [inline tests/test_sort_package_xml.py:21]: Where is `tmp_path` from?

Comment (KeisukeShima) [reply tests/test_sort_package_xml.py:21]: tmp_path is https://docs.pytest.org/en/6.2.x/tmpdir.html

Comment (kenji-miyake) [reply tests/test_sort_package_xml.py:21]: I see, looks nice. :+1:

**Review by kenji-miyake — APPROVED** [review]
Thank you!

**Pull request merged**